### DISCUSSION TITLE
Suggestion for english translators - avoid repetition

### DISCUSSION THREAD

**Discussion by raspin0** [linuxdeepin/developer-center#3344] [Features Request & Ideas | 特性请求 & 头脑风暴]
I have found two places where repetitions make UI look worse and harder to navigate. 

![Bez nazwy](https://user-images.githubusercontent.com/39202451/182170687-e67fb66d-858d-4509-b56b-fcf271551955.png)
![Bez nazwy](https://user-images.githubusercontent.com/39202451/182172684-5a54ca82-92dc-4f79-8f45-272318823a61.png)

**Comment by BLumia** [comment]
@meiyixiang 文案建议：如图所示。
